integration_go.mod

```diff
@@ -0,0 +1,91 @@
+module gopkg.in/DataDog/dd-trace-go.v1
```

Comment:
nitpick:

Should we add a comment just saying that this is manually made/maintained and that new integrations should be added to it?